index.js

```diff
@@ -22,6 +23,10 @@ var toString = Function.prototype.toString;
 var functionNameMatch = /\s*function(?:\s|\s*\/\*[^(?:*\/)]+\*\/\s*)*([^\s\(\/]+)/;
 function getFuncName(aFunc) {
   var name = '';
+  if (typeof aFunc !== 'function') {
```

Comment:
Maybe just nitpicking, but we could avoid initializing name when this `if` gets executed:
```javascript
var name;
if (typeof aFunc !== 'function') {
  return null;
}

name = '';
```